packages/app-mobile/components/screens/Note/commands/attachFile.ts

```diff
@@ -44,7 +44,13 @@ export const runtime = (props: CommandRuntimeProps): CommandRuntime => {
 	};
 	const attachPhoto = async () => {
 		// the selection Limit should be specified. I think 200 is enough?
-		const response: ImagePickerResponse = await launchImageLibrary({ mediaType: 'photo', includeBase64: false, selectionLimit: 200 });
+		const response: ImagePickerResponse = await launchImageLibrary({
+			mediaType: 'mixed',
+			videoQuality: 'low',
```

Comment:
What does the `videoQuality` do exactly? Does it convert the video before attaching it? And is it not possible to attach the video without any conversion?

Reply:
From [the documentation](https://github.com/react-native-image-picker/react-native-image-picker?tab=readme-ov-file#options):
> low, medium, or high on iOS, low or high on Android.

Based on the code, the `videoQuality` is converted to `UIImagePickerControllerQualityTypeLow`, UIImagePickerControllerQualityTypeMedium`, or `UIImagePickerControllerQualityTypeHigh`, then [passed to the system image picker](https://github.com/react-native-image-picker/react-native-image-picker/blob/ae1452e273ce0ff3adf1acbd21184de571d12c4f/ios/ImagePickerUtils.mm#L15).

"Low" was selected here with the goal of helping avoid sync issues caused by large attachments.

Reply:
Ok I guess that makes sense to convert to low quality as indeed that could help prevent sync issues